examples/dotnet/ShiftSchedulingSat.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+	<LangVersion>7.2</LangVersion>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+	<EnableDefaultItems>false</EnableDefaultItems>
+    <RestoreSources>../../packages;$(RestoreSources);https://api.nuget.org/v3/index.json</RestoreSources>
+  </PropertyGroup>
+  
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugType>full</DebugType>
+    <Optimize>true</Optimize>
+    <GenerateTailCalls>true</GenerateTailCalls>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <Compile Include="ShiftSchedulingSat.cs" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="Google.OrTools" Version="7.0.*" />
```

Comment:
please use `7.0-*`  otherwise it won't work with `7.0.XXXX-beta` incoming release.
you can also use (on linux) tools/generate_all_dotnet_proj.sh